src/modules/settings/index.js

```diff
@@ -1,5 +1,9 @@
 import {PlatformTypes} from '../../constants.js';
 import {loadModuleForPlatforms} from '../../utils/modules.js';
-import SettingsModule from './Settings.jsx';
+import TwitchSettingsModule from './twitch/Settings.jsx';
+import YoutubeSettingModule from './youtube/Settings.jsx';
 
-export default loadModuleForPlatforms([PlatformTypes.TWITCH, () => new SettingsModule()]);
+export default loadModuleForPlatforms(
```

Comment:
isn't the `Settings.jsx` file already doing this?

Reply:
?